packages/core/src/repositories/commitsRepository/index.ts

```diff
@@ -1,22 +1,19 @@
 import { DocumentVersion, Project } from '$core/browser'
+import { database } from '$core/client'
 import {
   CommitStatus,
   HEAD_COMMIT,
   ModifiedDocumentType,
 } from '$core/constants'
-import { NotFoundError, Result } from '$core/lib'
-import { commits, projects } from '$core/schema'
-import { recomputeChanges, RecomputedChanges } from '$core/services'
-import { assertCommitIsDraft } from '$core/services/documents/utils'
+import { InferedReturnType, NotFoundError, Result } from '$core/lib'
+import { assertCommitIsDraft } from '$core/lib/assertCommitIsDraft'
+import { buildCommitsScope } from '$core/repositories/commitsRepository/utils/buildCommitsScope'
+import { getHeadCommitForProject } from '$core/repositories/commitsRepository/utils/getHeadCommit'
```

Comment:
These changes were done because when doing the rollup it complained about circular dependencies. Although I don't need to do the build of core I would keep these changes because at the end I managed to generate a clean rollup build without Circular dependencies warnings.